### DISCUSSION TITLE
Different day / night ADU targets

### DISCUSSION THREAD

**Discussion by StarGeezerPhil** [aaronwmorris/indi-allsky#598] [Ideas]
@aaronwmorris, just wondering whether it would be possible to set different day and night ADU targets or if there's another solution to this...

I find my daylight images very dark, whereas it'd nice for these to be brighter for daytime timelapses etc.

Is there another way to achieve this - without making night images too bright?

**Comment by aaronwmorris** [comment]
Merged #619 with separate day and night ADU targets.

**Reply by StarGeezerPhil** [reply]
You are a star Aaron 🔥
